recaptcha/CTL000179.yaml

```diff
@@ -1,9 +1,9 @@
 bioguide: CTL000179
 contact_form:
   method: post
-  action: 
+  action:
   steps:
-    - visit: "http://cthousegop.com/contact-gail/"
+    - visit: "http://www.cthousegop.com/lavielle/contact-me/"
```

Comment:
In the future, if a target has recaptcha V2, you don't need to fix up the yaml - just move it to the recaptcha folder. It's fine to leave this though.